lib/ClangImporter/ClangImporter.cpp

```diff
@@ -2407,6 +2413,12 @@ ModuleDecl *ClangImporter::Implementation::loadModuleClang(
   // If we're asked to import the top-level module then we're done here.
   auto *topSwiftModule = finishLoadingClangModule(clangModule, importLoc);
   if (path.size() == 1) {
+    if (SwiftContext.LangOpts.EnableModuleLoadingRemarks)
+      SwiftContext.Diags.diagnose(SourceLoc(),
+                                  diag::module_loading_clang_finished,
```

Comment:
I think this may provide duplicate information to what's emitted here:
https://github.com/swiftlang/swift/blob/bb67d1ef0aba3470b6b31bb34651b8adaae56822/lib/AST/ASTContext.cpp#L2809

e.g. from a simple compile I already see remarks like:
```
<unknown>:0: remark: loaded module 'SwiftShims'; source: '/.../lib/swift/shims/module.modulemap', loaded: '/.../ModuleCache/SwiftShims-7XBJYH4ZCALSCKB881Z77AY2L.pcm'
```

Reply:
Could you please also check the results you get with `-Xcc -Rmodule-import`? I think that may also already emit some remarks that highlight what other module dependencies get brought in when importing a given Clang module. 

Though I do like the idea of emitting a list of all transitively-imported modules for a top-level import from Swift.